src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -27,4 +27,38 @@ public Integer getLength() {
     public String getColor() {
         return color;
     }
+
+    @Override
+    public boolean equals(Object object) {
+        if (this == object) {
+            return true;
+        }
+        if (object == null) {
+            return false;
+        }
+        if (object instanceof Rectangle) {
```

Comment:
why have you choose to use instanceof instead of getClass()?

Reply:
done